doc/Language/variables.pod6

```diff
@@ -637,20 +636,20 @@ dangerous.
 
 =head3 The C<$> Variable
 
-As well as explicitly declared named state variables C<$> can be used
+Beside explicitly declared named state variables, C<$> can be used
```

Comment:
You're right.
How about:
In addition to explicitly declared named state variables, C<$> can be used...